nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+app.use("/numbers", router);
+router.get("/add", async (request, response) => {
+    const first = parseInt(request.query.first);
+    const second = parseInt(request.query.second);
+    const addtwonumbers = first + second;
+    response.send(`Sum of Two numbers is:${addtwonumbers}`)
+})
+router.get("/multiply/:first/:second", async (request, response) => {
+    const first = parseInt(request.params.first);
```

Comment:
It's good to check the user inputs and return an error if the request has invalid inputs.
![image](https://user-images.githubusercontent.com/3575408/140639470-1b9e7306-b341-4c29-8db2-135217a4e8df.png)